pkg/app/sensor/execution/standalone.go

```diff
@@ -39,7 +39,7 @@ func NewStandalone(
 		)
 	}
 
-	eventFile, err := os.OpenFile(eventFileName, os.O_APPEND|os.O_WRONLY, 0644)
+	eventFile, err := os.OpenFile(eventFileName, os.O_APPEND|os.O_WRONLY|os.O_SYNC, 0644)
```

Comment:
This change is needed to avoid an awkward `exe.Close()` call before dumping the artifacts.